js/rendererjs/disabledplugin.js

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react'
+
+const containerStyle = {
+	display: 'flex',
+	alignItems: 'center',
+	justifyContent: 'center',
+	flexDirection: 'column',
+	backgroundColor: '#C6C6C6',
+	width: '100%',
+	height: '100%',
+}
```

Comment:
Probably.  I used inline css here because this component is mounted directly in the `body` of the plugins, and should be styled independently of the plugin's styling.